src/libs/ReactNativeLauncher.js

```diff
@@ -159,7 +159,8 @@ class ReactNativeLauncher extends Launcher {
       await this.stop()
     } catch (err) {
       log.error(err, 'start error')
-      await this.ensureAccountTriggerAndLaunch() // to create the job even if the error was raised before sendLoginSuccess
+      // to create the job even if the error was raised before sendLoginSuccess
+      await this.ensureAccountTriggerAndLaunch()
```

Comment:
What if the `ensureAccountTriggerAndLaunch` from line 149 is called before the throw? Will the second call produce conflicts?